packages/grid/x-data-grid/src/locales/frFR.ts

```diff
@@ -53,13 +53,13 @@ const frFRGrid: Partial<GridLocaleText> = {
   filterPanelOperator: 'Opérateur',
   filterPanelOperatorAnd: 'Et',
   filterPanelOperatorOr: 'Ou',
-  filterPanelColumns: 'Colonnes',
+  filterPanelColumns: 'Colonne',
```

Comment:
Here, the singular seems to be the right choice unless there will be multi-columns filters in the future?

Reply:
> there will be multi-columns filters in the future

multi-column is already supported in pro plan

```suggestion
  filterPanelColumns: 'Colonnes',
```